packages/@ourworldindata/components/src/TextWrap/TextWrap.tsx

```diff
@@ -194,16 +205,37 @@ export class TextWrap {
         else return lines
     }
 
+    @computed get lineCount(): number {
+        return this.lines.length
+    }
+
+    @computed get singleLineHeight(): number {
+        return this.fontSize * this.lineHeight
+    }
+
     @computed get height(): number {
-        const { lines, lineHeight, fontSize } = this
-        if (lines.length === 0) return 0
-        return lines.length * lineHeight * fontSize
+        if (this.lineCount === 0) return 0
+        return this.lineCount * this.singleLineHeight
+    }
+
+    @computed get heightWithoutOffsetedLine(): number {
```

Comment:
[The past form of offset is also offset](https://dictionary.cambridge.org/dictionary/english/offset) :)
```suggestion
    @computed get heightWithoutOffsetLine(): number {
```